Mathlib/CategoryTheory/Products/Basic.lean

```diff
@@ -352,4 +353,25 @@ def functorProdFunctorEquiv : (A ⥤ B) × (A ⥤ C) ≌ A ⥤ B × C :=
     counitIso := functorProdFunctorEquivCounitIso A B C, }
 #align category_theory.functor_prod_functor_equiv CategoryTheory.functorProdFunctorEquiv
 
+section Opposite
+
+open Opposite
+
+/-- The equivalence between the opposite of a product and the product of the opposites. -/
+@[simps]
+def prodOp : (C × D)ᵒᵖ ≌ Cᵒᵖ × Dᵒᵖ where
+  functor :=
+    { obj := λ X => ⟨op X.unop.1, op X.unop.2⟩,
+      map := λ f => ⟨f.unop.1.op, f.unop.2.op⟩ }
+  inverse :=
+    {
+      obj := λ ⟨X,Y⟩ => op ⟨X.unop, Y.unop⟩,
+      map := λ ⟨f,g⟩ => op ⟨f.unop, g.unop⟩ }
+  unitIso := by aesop_cat
+  counitIso := by aesop_cat
```

Comment:
```suggestion
  unitIso := Iso.refl _
  counitIso := Iso.refl _
```
Even though `aesop(_cat)` can do some magic, we should refrain from creating data from these tactics!